python/python/bystro/ancestry/ancestry_types.py

```diff
@@ -122,7 +122,6 @@ class PopulationVector:
     JPT: ProbabilityInterval = field(validator=ProbIntValidator)
     KHV: ProbabilityInterval = field(validator=ProbIntValidator)
     LWK: ProbabilityInterval = field(validator=ProbIntValidator)
-    MAG: ProbabilityInterval = field(validator=ProbIntValidator)
```

Comment:
Is MAG being removed because we're no longer expecting it in any datasets or is it being removed here for a different reason?

Reply:
no longer expecting it in any datasets